packages/emails/email-workflow.ts

```diff
@@ -0,0 +1,288 @@
+// eslint-disable-next-line no-restricted-imports
+import { proxyActivities } from "@temporalio/workflow";
+import type { TFunction } from "next-i18next";
+
+import type { EventNameObjectType } from "@calcom/core/event";
+import type * as activities from "@calcom/emails/email-manager";
+import type { CalendarEvent, Person } from "@calcom/types/Calendar";
+
+import type { MonthlyDigestEmailData } from "./src/templates/MonthlyDigestEmail";
+import type { EmailVerifyLink } from "./templates/account-verify-email";
+import type { OrganizationNotification } from "./templates/admin-organization-notification";
+import type { EmailVerifyCode } from "./templates/attendee-verify-email";
+import type { Feedback } from "./templates/feedback-email";
+import type { PasswordReset } from "./templates/forgot-password-email";
+import type { OrgAutoInvite } from "./templates/org-auto-join-invite";
+import type { OrganizationEmailVerify } from "./templates/organization-email-verification";
+import type { TeamInvite } from "./templates/team-invite-email";
+
+const emailActivities = proxyActivities<typeof activities>({
+  startToCloseTimeout: "3 minutes",
+  retry: {
+    // nonRetryableErrorTypes: ["@calcom/emails/EmailError"],
```

Comment:
We can choose to define/configure errors that shouldn't retry a failed task (Activity)